<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>1052</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="managerTab">
       <attribute name="title">
        <string>Managers</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Nombre</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="name"/>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Direcci칩n</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="address"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>DPI</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="dpi"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="insertManager">
             <property name="text">
              <string>Guardar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="2" column="0" colspan="2">
            <widget class="QTextEdit" name="manager_logs"/>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QTableWidget" name="manager_table">
             <column>
              <property name="text">
               <string>Nombre</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Direcci칩n</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>DPI</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="proyectTab">
       <attribute name="title">
        <string>Proyecto</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QFrame" name="frame_4">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Nombre</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="project_name"/>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Presupuesto (Q)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="budget"/>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Manager</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="manager_id"/>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Servicio</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="service_id"/>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Familia beneficiada</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="3" column="0">
               <widget class="QLineEdit" name="family_address"/>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_19">
                <property name="text">
                 <string>Ingresos</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLineEdit" name="monthly_income"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>Apellido</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>Direcci칩n</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLineEdit" name="last_name"/>
              </item>
              <item row="6" column="0">
               <widget class="QFrame" name="frame_10">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="QLabel" name="label_12">
                   <property name="text">
                    <string>Nombre</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="sibling_name"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_16">
                   <property name="text">
                    <string>Edad</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="age"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="appendSibling">
                   <property name="text">
                    <string>A침adir</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QListView" name="member_list"/>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFrame" name="frame_5">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="insertProject">
             <property name="text">
              <string>Guardar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QFrame" name="frame_6">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="0" colspan="2">
            <widget class="QTableWidget" name="project_table">
             <column>
              <property name="text">
               <string>Nombre</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Presupuesto (Q)</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Manager</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Servicio</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Familia beneficiada</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QTextEdit" name="project_logs"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1019</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
